### ISSUE TITLE
(2.0) Keyboard: Quicksearch filenames and brush resize shortcuts don't work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [birdal46/grafx2#180]
```
In the file selectors for Save/load, typing letters to find the closest 
filename has no effect.

When resizing a brush, keys n d x y shift-d shift-x shift-y are supposed 
to assign sizes 'normal', 'double', 'half' etc. They have no effect.

---

This is a regression in 2.0 compared to b99%. When I tried to solve the 
crashes on win98, I enabled Unicode only while entering text (instead 
of "always") (r817), and I forgot that the two functions above rely on 
Key_ANSI; and Key_ANSI cannot be determined if Unicode is disabled.

The fix is to remove the enabling/disabling of Unicode in Readline, and 
put it back in program initialization (main.c) instead.

I'm just not sure if we should do it on the 2.0 branch, or put it in 2.1 
whenever it's released.
```

Original issue reported on code.google.com by `yrizoud` on 17 Jun 2009 at 1:50

**Comment by GoogleCodeExporter** [comment]
```
Mh... just try to avoid a different build for 98vsXP if possible :)

You can revert to b99 behaviour in 2.0 if you think it makes sense, but it 
would be 
nice to still avoid the crash in 2.1. If it's not possible, then we'll have to 
drop 
win98 support. As you wish, I don't need win98...
It's better to have some broken shortcuts than a crashing app, no ?

(also, please use labels to tag your issues, "Milestone-2.0" is still there :))
```

Original comment by `pulkoma...@gmail.com` on 17 Jun 2009 at 3:40
- Added labels: **Milestone-2.0**, **OpSys-Windows**
- Removed labels: **OpSys-All**

**Comment by GoogleCodeExporter** [comment]
```
Did you forget my only dev machine is running win98 ;)
Don't worry, the real fix for the win98-non-qwerty crash has no bad effects on 
any 
XP machines I tested. Fixing this current issue will not bring back the win98 
crashes.

```

Original comment by `yrizoud` on 17 Jun 2009 at 4:24

**Comment by GoogleCodeExporter** [comment]
```
Ok, then it can be fixed on 2.0, as it's really a bug :) (ie. we say it works 
in the 
documentation)
```

Original comment by `pulkoma...@gmail.com` on 17 Jun 2009 at 4:27

**Comment by GoogleCodeExporter** [comment]
```
fixed in r875
```

Original comment by `pulkoma...@gmail.com` on 18 Jun 2009 at 9:08
- Changed state: **Fixed**
